pumpkin-protocol/src/client/play/c_command_suggestions.rs

```diff
@@ -45,12 +45,12 @@ impl<'a> ClientPacket for CCommandSuggestions<'a> {
 
 #[derive(PartialEq, Eq, Hash, Debug)]
 pub struct CommandSuggestion<'a> {
-    pub suggestion: &'a str,
+    pub suggestion: String,
```

Comment:
why did you make this a String ?

Reply:
I had lifetime issues in the command suggestion because the str variable would be dropped after the function.
Could this be implemented a other way?